pages/common/git-var.md

```diff
@@ -0,0 +1,12 @@
+# git var
+
+> Prints a Git logical variable.
```

Comment:
Perhaps this will read better?
```suggestion
> Prints the value of a Git logical variable.
```

Reply:
Yep.  I copied it right from https://git-scm.com/docs/git-var.

Reply:
Actually I like this better:
```diff
-Prints a Git logical variable's value.
+Prints the value of a Git logical variable.
```
WDYT, @sbrl?